server/pkg/asset/preview_type.go

```diff
@@ -25,6 +26,8 @@ func PreviewTypeFrom(p string) (PreviewType, bool) {
 		return PreviewTypeGEO3D, true
 	case PreviewTypeMODEL3D:
 		return PreviewTypeMODEL3D, true
+	case PreviewTypeUNKNOWN:
+		return PreviewTypeUNKNOWN, true
 	default:
 		return PreviewType(""), false
 	}
```

Comment:
Use `strings.ToLower` for `p` in this method

Reply:
I have already deleted this method since it is not used

Reply:
https://github.com/reearth/reearth-cms/blob/main/server/internal/adapter/gql/resolver_mutation_asset.go#L42

This line occurs a bug that GraphQL representation is uppercase, but domain model's representation is lowercase. So we need a function to convert GraphQL's asset preview type to domain model's asset preview type.

Reply:
I thought that this is happening in the adapter layer by this function:
https://github.com/reearth/reearth-cms/blob/feat/add-unknown-preview-type/server/internal/adapter/gql/gqlmodel/convert_asset.go#L32

Reply:
It is domain model -> graphql model, but I say about graphql mode -> domain model.

Reply:
Done!